app/liquid/views/partials/_form.liquid

```diff
@@ -16,7 +16,8 @@
           <input type="tel" name="{{field.name}}" class="form-control form__content" id="" placeholder="{{ field.label }}" />
         {% when 'checkbox' %}
           <label class="checkbox-label">
-            <input type="checkbox" name="{{field.name}}" class="form__content" /> {{ field.label }}
+            <input name="{{field.name}}" type="hidden" value="0" />
```

Comment:
http://apidock.com/rails/ActionView/Helpers/FormHelper/check_box

> The hidden field has the same name and its attributes mimic an unchecked check box. This way, the client either sends only the hidden field (representing the check box is unchecked), or both fields. Since the HTML specification says key/value pairs have to be sent in the same order they appear in the form, and parameters extraction gets the last occurrence of any repeated key in the query string, that works for ordinary forms.

Otherwise we don't a value when the checkbox is empty.